download.go

```diff
@@ -103,6 +106,33 @@ func (app *Application) VideoDownload(w http.ResponseWriter, r *http.Request) {
 	case TypeVideo:
 		if r.Method != "HEAD" {
 			videoDownloadCounter.Inc()
+			tx, err_db := app.db.Begin()
+
+			if err_db != nil {
+				log.Println("cannot start transaction (downloads) ", err_db)
```

Comment:
please make use of the new loguru format

```suggestion
				log.WithFields(log.Fields{"error": err}).Error("cannot start transaction (downloads)")
```